skada/metrics.py

```diff
@@ -370,3 +396,24 @@ def _score(self, estimator, X, y, sample_domain=None, **kwargs):
         var_w = np.var(weights, ddof=1)
         eta = -cov / var_w
         return self._sign * (np.mean(weighted_error) + eta * np.mean(weights) - eta)
+    
+
+    def cross_entropy_loss(self, y_true, y_pred, epsilon=1e-15):
```

Comment:
In our case yes because we are computing the CE loss for each sample individually.
The **sklearn** implementation throws an error if at least one of each possible label is not present in `y_pred`.
`Error: ValueError: The number of classes in labels is different from that in y_pred.`